### PR TITLE
Add config docs

### PR DESCRIPTION
**Pull request by charlie0220**
Signed-off-by: Charles Huang charlie850220@gmail.com

Add config docs of datacatalog and scheduler and fix the script.

related to [#1849](https://github.com/flyteorg/flyte/issues/1849)

docs: https://flyte--2018.org.readthedocs.build/en/2018/deployment/cluster_config/index.html#deployment-cluster-config

### CODE REVIEW

File: rsts/deployment/cluster_config/datacatalog_config.rst [hunk 1]
```diff
@@ -0,0 +1,688 @@
+.. _datacatalog-config-specification:
```
[the remark this answers is not in the document]
Comment (pingsutw) [reply rsts/deployment/cluster_config/datacatalog_config.rst:1]: ditto

File: rsts/deployment/cluster_config/scheduler_config.rst [hunk 2]
```diff
@@ -0,0 +1,1964 @@
+.. _scheduler-config-specification:
```
[the remark this answers is not in the document]
Comment (charlie0220) [reply rsts/deployment/cluster_config/scheduler_config.rst:1]: Should file name change to flytescheduler_config.rst?
Which link do you refer to?

[the remark this answers is not in the document]
Comment (pingsutw) [reply rsts/deployment/cluster_config/scheduler_config.rst:1]: image

**Comment by EngHabu** [comment]
@charlie0220 @pingsutw is this ready for another round of reviews?

**Comment by pingsutw** [comment]
@EngHabu yes, thanks.

**Review by kumare3 — APPROVED** [review]

**Pull request merged**